src/util/FormatTime.cpp

```diff
@@ -71,4 +161,49 @@ QString formatTime(std::chrono::seconds totalSeconds)
             std::numeric_limits<int>::max()))));
 }
 
+QString formatLongFriendlyDuration(const QDateTime &from, const QDateTime &to)
```

Comment:
As suggested by @Mm2PL in chat, it would be cool if it would use the "4 oldest" units instead

e.g.
1 year 1 month 1 day 1 hour ago
1 month 1 day 1 hour 1 minute ago
1 hour 1 minute ago